azurelinuxagent/ga/agent_update_handler.py

```diff
@@ -78,12 +90,35 @@ def _get_daemon_version_for_update():
         # use the min version as 2.2.53 as we started setting the daemon version starting 2.2.53.
         return FlexibleVersion("2.2.53")
 
+    @staticmethod
+    def _get_initial_update_state_file():
+        """
+        This file keeps if initial update is attempted or not
+        """
+        return os.path.join(conf.get_lib_dir(), INITIAL_UPDATE_STATE_FILE)
+
+    def _save_initial_update_state(self):
```

Comment:
_get_initial_update_state_file & _save_initial_update_state: one is "file", the other "state". we should use the same naming convention, probably "file"